src/status_im/ui/components/animation.cljs

```diff
@@ -36,7 +36,7 @@
   (.stopAnimation anim-value))
 
 (defn value [anim-value]
-  (.-value anim-value))
+  (.-_value anim-value))
```

Comment:
could you please elaborate on this change?

Reply:
Previous method does not work, check out stackoverflow [here](https://stackoverflow.com/questions/41932345/get-current-value-of-animated-value-react-native)

Reply:
yeah it's not idiomatic to read value we should try to avoid this

Reply:
Do you suggest delete this function? Otherwise the previous function will make no sense.